elixir/apps/api/lib/api/gateway/channel.ex

```diff
@@ -150,7 +151,8 @@ defmodule API.Gateway.Channel do
             flow_id: flow_id,
             resource: Views.Resource.render(resource),
             expires_at: DateTime.to_unix(authorization_expires_at, :second),
-            payload: payload
+            payload: payload,
+            client: client
```

Comment:
This is backwards-compatible because old gateways will simply ignore this field and we don't remove any existing ones.

Reply:
@thomaseizinger this looks good, all we need is to check in tests that this field is present (you can also copy that part from the other test), just to make sure we won't break this in future.

Reply:
So I think this needs to use `Views.render(client)`, but then that will be expecting the preshared_key.